src/files/overlay/Overlay.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Modal } from 'react-overlays'
+
+function Overlay ({children, show, onLeave, className, ...props}) {
+  return (
+    <Modal
+      {...props}
+      show={show}
+      className={`${className} fixed top-0 left-0 right-0 bottom-0 z-max flex items-center justify-around`}
+      backdropClassName='fixed top-0 left-0 right-0 bottom-0 bg-black o-50'
+      onBackdropClick={onLeave}
+      onEscapeKeyDown={onLeave}>
+      {children}
+    </Modal>
+  )
+}
+
+Overlay.propTypes = {
+  show: PropTypes.bool.isRequired,
+  onLeave: PropTypes.func.isRequired
```

Comment:
I don't think this should be required. I imagine there'll be cases when we just want to close the modal without any other action!

Reply:
Hmm, so how would you change the `show` prop? If you don't have a `onLeave` trigger, you don't know the user wants to close the modal. And that way you can't change the `show` prop to `false`.

Reply:
You're right, my bad!

Don't know why, but I was thinking that the `onLeave` was from `react-overlay` and that it hid the modal by default!